geometry_msgs/Vector3 body_position
geometry_msgs/Vector3 body_rotation
geometry_msgs/Point front_right_leg
geometry_msgs/Point front_left_leg
geometry_msgs/Point back_left_leg
geometry_msgs/Point back_right_leg
---
black_mouth_kinematics/LegJoints front_right_leg
black_mouth_kinematics/LegJoints front_left_leg
black_mouth_kinematics/LegJoints back_left_leg
black_mouth_kinematics/LegJoints back_right_leg